interpretation/ww_head.py

```diff
@@ -203,7 +205,8 @@ def process(message, isGameMaster = False):
         '''myrole'''
         # This command sends the user's role back to them in a DM.
         if is_command(message,['myrole']):
-            return [Mailbox().dm("Your role is **{}**.".format(db.db_get(message.author.id,'role')), message.author.id)]
+            return [Mailbox().dm("Your role is **{}**.".format(db_get(message.author.id,'role')), message.author.id,False,[db_get(message.author.id,'emoji')])]
+            print("This should not execute!")
```

Comment:
This is for debugging. I guess we could leave it out, as it is quite un-informative and needless, but hey, should I really spend a whole commit on that little change? 🤷‍♀️

Reply:
Yes, yes you should.